common/lib/common-utils/tsconfig.json

```diff
@@ -6,9 +6,13 @@
     ],
     "compilerOptions": {
         "rootDir": "./src",
-        "outDir": "./dist"
+        "outDir": "./dist",
+        "baseUrl": ".",
+        "paths": {
+            "perf_hooks": ["types/perf_hooks.d.ts"]
```

Comment:
Explicitly pointing perf_hooks typing to `types` folder to explicitly opt-in individual node modules.